src/host/premake.h

```diff
@@ -38,12 +38,15 @@
 #elif defined (__GNU__)
 #define PLATFORM_HURD  (1)
 #define PLATFORM_OS  "hurd"
+#elif defined (__COSMOPOLITAN__)
+#define PLATFORM_COSMO  (1)
+#define PLATFORM_OS  "cosmopolitan"
```

Comment:
Won't this mean the default target OS will be this instead of the users host OS?